firebase-firestore/src/commonTest/kotlin/dev/gitlive/firebase/firestore/firestore.kt

```diff
@@ -100,7 +100,11 @@ class FirebaseFirestoreTest {
 
         firestore = Firebase.firestore(app).apply {
             useEmulator(emulatorHost, 8080)
-            setSettings(persistenceEnabled = false)
+            settings = firestoreSettings(settings) {
+                cacheSettings = memoryCacheSettings {
+                    gcSettings = memoryEagerGcSettings { }
+                }
+            }
```

Comment:
this syntax is bizarre, why not use the same pattern you used for encodeSettings? namely  `setSettings { ... }`  and put all the setting functions in the builderscope instead of at the package level

Reply:
Oh shit 🤣

Reply:
I'm starting to get the impression they have less kotlin experience on the android sdk team that us 😅